source/index.html.md

```diff
@@ -167,6 +169,168 @@ HttpWebResponse response = (HttpWebResponse)request.GetResponse();
 
 ```
 
+```java
+import java.io.File;
+import java.io.FileWriter;
+import kong.unirest.Unirest;
+import kong.unirest.HttpResponse;
+
+public class recognize {
+    public static void main(String[] args){
+        String token = "MY_API_KEY";
+        String file = "C:\\assets\\demo.jpg";
+        
+        try{
+            HttpResponse<String> response = Unirest.post("https://api.platerecognizer.com/v1/plate-reader/")
+            .header("Authorization", "Token "+token)
+            .field("upload", new File(file))
+            .asString();
+            System.out.println("Recognize:");
+            System.out.println(response.getBody().toString());
+        }
+        catch(Exception e){
+            System.out.println(e);
+        }
+        
+        try{
+            HttpResponse<String> response = Unirest.get("https://api.platerecognizer.com/v1/statistics/")
+            .header("Authorization", "Token "+token)
+            .asString();
+            System.out.println("Usage:");
+            System.out.println(response.getBody().toString());
+        }
+        catch(Exception e){
+            System.out.println(e);
+        }
+    }
+}
+```
+
+```cpp
+#define CURL_STATICLIB
+#include "curl/curl.h"
+#include <iostream>
+#include <string>
+#include "json/json/json.h"
+#include "json/jsoncpp.cpp"
+#include <fstream>
+
+
+
+using namespace std;
+
+namespace
+{
+		size_t callback(
+		const char* in,
+		std::size_t size,
+		std::size_t num,
+		std::string* out)
+	{
+		const size_t totalBytes(size * num);
+		out->clear();
+		out->append(in, totalBytes);
+		return totalBytes;
+	}
+}
+
+Json::Value sendRequest(string auth_token, string fileName) {
+	
+	curl_global_init(CURL_GLOBAL_ALL);
+
+	
+	curl_off_t speed_upload, total_time;
+	curl_mime *form = NULL;
+	curl_mimepart *field = NULL;
+
+	CURL *hnd = curl_easy_init();
+
+	//curl_easy_setopt(hnd, CURLOPT_CUSTOMREQUEST, "POST");
+	curl_easy_setopt(hnd, CURLOPT_URL, "https://api.platerecognizer.com/v1/plate-reader/");
+
+	form = curl_mime_init(hnd); //initialize form fields
+
+	/* Fill in the file upload field */
+	field = curl_mime_addpart(form);
+	curl_mime_name(field, "upload");
+	curl_mime_filedata(field, fileName.c_str());
+	curl_easy_setopt(hnd, CURLOPT_MIMEPOST, form);
```

Comment:
Just include the selected lines.
Add a comment with a link to the full source code.